library/proc_macro/src/lib.rs

```diff
@@ -815,6 +815,16 @@ pub enum Delimiter {
     /// "macro variable" `$var`. It is important to preserve operator priorities in cases like
     /// `$var * 3` where `$var` is `1 + 2`.
     /// Invisible delimiters might not survive roundtrip of a token stream through a string.
+    ///
+    /// <div class="warning">
+    ///
+    /// Note: rustc currently (1.77 and below) ignores the grouping of tokens
```

Comment:
If the correct current version cannot be inserted here automatically, then it should probably not be mentioned at all.
Otherwise it will either always be outdated, or require regular updates.

Reply:
Possibly, we could reference #67062?  Then if we forget to update the docs, the reader who follows the link will see a closed ticket.

Reply:
Fair enough, I'll remove the version mention. #67062 is already mentioned a few lines later. I'll just remove the version statement, and just make a comment in #67062 to remove this warning if this actually ever gets fixed.